File: Herencia/gato.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   gato.cpp
 * Author: Usuario
 * 
 * Created on 4 de octubre de 2017, 23:26
 */

#include "gato.h"

gato::gato() {
}

gato::gato(string raza,string nombre) {
    this->raza=raza;
    
}

string gato::getRaza() {
    return this->raza;
}

void gato::setRaza(string raza) {
    this->raza=raza;
}



gato::~gato() {
}

File: ayudantia7_2019/src/ayudantia7_2019/Gato.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ayudantia7_2019;

/**
 *
 * @author Juan Pablo Martinez
 */
public class Gato extends Mascota {
    private int years;

    public Gato(int years, String nombre, int peso) {
        super(nombre, peso);
        this.years = years;
    }

    @Override
    public int calcularAlimento() {
        int cant = years *100;
        return cant;
    }
    
}
